config/api.yml

```diff
@@ -929,17 +929,27 @@
     - :collection
     :verbs: *gp
     :klass: MiqRequest
+    :subcollections:
+    - :request_tasks
+    - :tasks
     :collection_actions:
       :get:
       - :name: read
         :identifier: miq_request_show_list
       :post:
       - :name: query
         :identifier: miq_request_show_list
+      - :name: create
+        :identifier: vm_miq_request_new
```

Comment:
yes, this needs to change. Couldn't fine a generic miq_request_new role identifier.  Part of the TODO is to support different identifier based on request_type.  One option is to open the door here (no role check), but enforce it in the create method based on the type.